REST_API.md

```diff
@@ -183,9 +186,15 @@ The following synonyms and shortened values for standard operations are supporte
 - `delete` - `d`
 - `manage` (any operation is allowed) - `m`, `admin`
 
+`include-patterns` is an optional argument (effectively is `["**"]` by default) 
+which allows specifying path patterns to which permissions are applied.
```

Comment:
@olegmoz I'd suggest to rephrase:
`include-patterns` is an optional argument (effectively value `["**"]` is set by default) 
which allows specifying path patterns to apply permissions to.